src/EditorFeatures/CSharpTest/AutomaticCompletion/AutomaticLineEnderTests.cs

```diff
@@ -1755,6 +1755,59 @@ public class Foo
             Test(secondResult, firstResult);
         }
 
+        [WpfFact]
+        public void TestObjectCreationExpressionInUsingStatement()
+        {
+            var initialMarkup = @"
+public class Bar
+{
+    public void M()
+    {
+        using(var a = n$$ew F$$oo($$)$$)
+    }
+}
+public class Foo
+{
+    public int HH { get; set; }
+    public int PP { get; set; }
+}";
+
+            var firstResult = @"
+public class Bar
+{
+    public void M()
+    {
+        using(var a = new Foo()
+        {
+            $$
+        })
+    }
+}
+public class Foo
+{
+    public int HH { get; set; }
+    public int PP { get; set; }
+}";
+
+            var secondResult = @"
+public class Bar
+{
+    public void M()
+    {
+        using(var a = new Foo())
+            $$
```

Comment:
This is one of the primary cases I think we're going to need to review once this feature is in the internal previews. Good to have a test for it so we are tracking the initial behavior.